dev-debug/gdb-dashboard/gdb-dashboard-0.17.2.ebuild

```diff
@@ -21,7 +21,10 @@ fi
 LICENSE="MIT"
 SLOT="0"
 
-RDEPEND="dev-debug/gdb[python]"
+RDEPEND="
+	dev-debug/gdb[python]
```

Comment:
I don't we can enable py3.13, if the `gdb` ebuilds don't support it yet.

Reply:
Ok, removed py3.13.